spyder/config/manager.py

```diff
@@ -641,9 +641,16 @@ def reset_shortcuts(self):
 try:
     CONF = ConfigurationManager()
 except Exception:
-    from qtpy.QtWidgets import QMessageBox
-    from spyder.app.utils import create_application
-    app = create_application()
+    from qtpy.QtWidgets import QApplication, QMessageBox
+
+    # Create QApplication in order to display the message below.
+    # NOTE: We don't use the functions we have to create a QApplication here
+    # because they could import CONF at some point, which would make this
+    # this fallback fail.
+    # See issue spyder-ide/spyder#17889
+    app = QApplication(['Spyder'])

```

Comment:
Should we add here a `QApplication.instance()` validation just in case @ccordoba12 ?

Reply:
Great point! I'll do the respective changes.